plugin/pkg/scheduler/factory/factory_test.go

```diff
@@ -36,6 +36,10 @@ import (
 	latestschedulerapi "k8s.io/kubernetes/plugin/pkg/scheduler/api/latest"
 )
 
+const (
+	SchedulerName = "default-scheduler"
```

Comment:
given the fact that we reference the same string in several locations could we coalesce on a single constant shared by all of them?  e.g. I counted 20 refs to "default-scheduler"